src/mlpack/tests/qdafn_test.cpp

```diff
@@ -155,18 +155,18 @@ BOOST_AUTO_TEST_CASE(SerializationTest)
   for (size_t i = 0; i < qdafn.NumProjections(); ++i)
   {
     BOOST_REQUIRE_EQUAL(qdafnXml.CandidateSet(i).n_rows,
-        qdafn.CandidateSet(i).n_rows);
+                        qdafn.CandidateSet(i).n_rows);
     BOOST_REQUIRE_EQUAL(qdafnText.CandidateSet(i).n_rows,
-        qdafn.CandidateSet(i).n_rows);
+                        qdafn.CandidateSet(i).n_rows);
     BOOST_REQUIRE_EQUAL(qdafnBinary.CandidateSet(i).n_rows,
-        qdafn.CandidateSet(i).n_rows);
+                        qdafn.CandidateSet(i).n_rows);
```

Comment:
hm, this looks odd, not sure, why it aligns the parameter, maybe because it's a macro?

Reply:
This was mentioned in the `Design Guidelines` under `Method Declaration`. By default, it should align in the above manner. The previous format was only for those cases where even the above format leads to an exceed on the 80 character line length.